### ISSUE TITLE
Rebasing with the orignal bsnes source

### ISSUE THREAD

**Issue by planetbrian** [Marat-Tanalin/bsnes-mt#24]
According to the author's website, they occasionally merge updates from the community bsnes repo, which has not seen activity since last October. While I am not certain what Mr. Tanalin defines as important changes, I believe it would serve this project well to merge over technical improvements made to the community repo.

**Comment by Marat-Tanalin** [comment]
Makes sense indeed. To be fair, Afaict, there were no changes in bsnes that affect emulation in a considerable way, since the last bsnes→bsnes-mt sync. Nonetheless, I plan to include at least some of the changes in a next bsnes-mt version together with some new unique bsnes-mt features, probably in the upcoming release going to happen when I have time, hopefully this summer or so.
